### ISSUE TITLE
Enable rpm/deb testing of non-x64 Linux packages

### ISSUE THREAD

**Issue by sxa** [adoptium/installer#375]
At the moment we do not have the support for running TestContainers on non-x64 platforms. We should look at implementing something that allows us to run the installer testing on the other architectures which we support.

**Comment by smlambert** [comment]
The year ago now that we talked with the TestContainers folks, there was a way to add support for non-x64 platforms.  Need to find the notes from that, something about changing Ryuk, but also its a year ago, and seems there are official updates that may help us: https://twitter.com/whichrich/status/1341730726270492672.

**Comment by sxa** [comment]
Yep, whether it's with "official" options (0bviously preferred if we can) or just something we knock up ourselves to do a basic install and `java -version` we should get something in place. Obviously not necessary for this first release as we can verify manually but has to be "on the list".

**Comment by jerboaa** [comment]
Once this gets enabled, those hard-coded values need to go: https://github.com/adoptium/installer/pull/379/files#diff-fdcc04743d05ab4da45b68037fc038027fbc92cb032e305ebc832642db38875aL37-L38

E.g. via an environment variable `ARCH` or some such.

**Comment by smlambert** [comment]
related: https://github.com/testcontainers/moby-ryuk/pull/21

**Comment by sophia-guo** [comment]
Just a note: 
1 . newly enabled sle12 may not be enable to add tests? https://github.com/adoptium/installer/pull/426/files
2. probably will be enabled in a short time RHEL6 https://github.com/adoptium/installer/pull/412/files

**Comment by karianna** [comment]
For those watching, linuxNew has been renamed to linux

**Comment by sxa** [comment]
It shouldn't be using `dockerBuild` as that's exclusively for hosts configured to run the dynamically generated docker images we use for building the JDK. `docker` should be a suitable one to use instead:

https://ci.adoptopenjdk.net/label/docker&&linux&&(s390x||armv7l)/
